website/layouts/_shortcodes/tabs.html

```
@@ -0,0 +1,82 @@
+    {{- $content = replace $content (printf "id=%q" $headingID) (printf "id=%q" $unique) -}}
+    {{- $seen = merge $seen (dict $unique $tab.Name) -}}
+  {{- else -}}
+    {{- $seen = merge $seen (dict $headingID $tab.Name) -}}
+  {{- end -}}
+{{- end }}
+{{- /* Collect headings from this tab into toc-extra, after dedup has finalised all IDs. */ -}}
+{{- $.Page.Scratch.Add "toc-extra" (slice (dict "type" "group" "name" $tab.Name)) -}}
+{{- range findRE `<h[2-6][^>]*id="[^"]+"` $content -}}
+  {{- $level := replaceRE `<h([2-6]).*` "$1" . | int -}}
+  {{- $id := replaceRE `.*\bid="([^"]+)".*` "$1" . -}}
+  {{- $textMatches := findRE (printf `<h[2-6][^>]*id="%s"[^>]*>[^<]+` $id) $content -}}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)